package com.youppix.ecommercecourse.domain.model.bottomBar

import androidx.compose.runtime.Immutable
import androidx.compose.ui.graphics.vector.ImageVector

@Immutable
data class BottomBar(
    val title : String ,
    val selectedIcon  : ImageVector ,
    val unselectedIcon : ImageVector ,
    val screen : Int
)